front-end/studio/src/app/pages/apis/{apiId}/editor/_components/forms/main/contact-section.component.html

```diff
@@ -1,7 +1,4 @@
-<section type="contact" label="CONTACT"
-         contextHelp="Update the information contained in this section to provide users of the API details about how to contact you should they need help."
-         collaborationNodePath="/info/contact"
-         [validationModels]="[contact()]">
+<section type="contact" label="CONTACT" contextHelp="Update the information contained in this section to provide users of the API details about how to contact you should they need help." collaborationNodePath="/info/contact" [validationModels]="[contact()]">
```

Comment:
Are there changes to this file or is it just reformatting?  If it's the latter could you revert this?

Reply:
Same question for the 'info' section below.

Reply:
Yes this is an issue with my IDE settings. I'll revert that one as well as the `info`section below.